squirrel_interaction/src/squirrel_interaction/board/serial_api.py

```diff
@@ -71,16 +84,41 @@ def move_to(self, motor, degrees):
         finally:
             self._mutex.release()
 
+    def start_motors(self):
+        """ Sets initial speeds for all Motors """
+	self.reset()
+        for motor in _MOTORS:
+            self.start_motor(motor)
+
+    def stop_motor(self, motor):
+        """ Stops motor [param 0] """
+        try:
```

Comment:
@HPovoa Should we acquire the lock here and in start_motors as well. Just to be safe.